src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -462,6 +462,28 @@ protected internal virtual DurableOrchestrationClient GetClient(OrchestrationCli
                 ExtendedSessionsEnabled = this.Options.ExtendedSessionsEnabled,
                 ExtendedSessionIdleTimeout = extendedSessionTimeout,
                 MaxQueuePollingInterval = this.Options.MaxQueuePollingInterval,
+                TrackingStoreStorageAccountDetails = this.GetStorageAccountDetailsOrNull(
+                    this.Options.TrackingStoreConnectionStringName),
+                TrackingStoreNamePrefix = this.Options.TrackingStoreNamePrefix,

```

Comment:
If `this.Options.TrackingStoreNamePrefix` is null, then we are overwriting the default value for `AzureStorageOrchestrationServiceSettings.TrackingStoreNamePrefix`, which is `"DurableTask"`. There are several options to fix this, but ideally we would do it without hardcoding "DurableTask" as the default value here as well, considering it is already in `DurableTask.AzureStorage`.